multi-node.sh

```diff
@@ -110,6 +110,15 @@ for IP in ${INSTANCE_IPS[@]}; do
 done
 wait
 
+for IP in ${INSTANCE_IPS[@]}; do
+    ssh -o ConnectTimeout=30 -o StrictHostKeyChecking=no -T -i ~/${slave_keypair} ${ami[1]}@${IP} \
```

Comment:
We can force a udev rule reload with udevadm instead of a reboot.

Reply:
I tried udev reload and trigger but it did not work. I added an environmental variable REBOOT_AFTER_INSTALL to control reboot.